templates/v-core-3-printer.template.cfg

```diff
@@ -109,6 +109,8 @@ position_endstop: 0
 [include config/extruders/bmg.cfg]
 # [include config/extruders/lgx.cfg]
 # [include config/extruders/orbiter.cfg]
+# [include config/extruders/orbiter.cfg]
```

Comment:
remove this line

Reply:
oops this should have been the `orbiter-1004.cfg`. Should I still delete it, or make it `orbiter-1004.cfg`?